controllers/provider-aws/pkg/controller/backupentry/actuator.go

```diff
@@ -53,5 +53,5 @@ func (a *actuator) Delete(ctx context.Context, be *extensionsv1alpha1.BackupEntr
 		return err
 	}
 
-	return awsClient.DeleteObjectsWithPrefix(ctx, be.Spec.BucketName, fmt.Sprintf("%s/", be.Name))
+	return awsClient.DeleteObjectsWithPrefix(ctx, be.Spec.BucketName, path.Join(be.Name, "/"))
```

Comment:
Is this change intended? It leads to different results:

* `fmt.Sprintf("%s/", be.Name)` results in `<name>/`
* `path.Join(be.Name, "/")` results in `<name>`

See https://play.golang.com/p/1thfsZeX5J0

Reply:
Ahh will revert that. Thanks for pointing that out.